python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -13,6 +13,9 @@
 
 logger = getLogger()
 
+_reg_id_str = '_registry_id'
+_id_str = 'id'
+
```

Comment:
I'm not a big fan of this technique of putting attribute names in string and using `getattr` everywhere. Unless we're planning on changing the attribute name it's solving a non-existent problem. It's a level of indirection which I'm not sure is worth it given the noise it adds to the code.

I'm saying that we should change the code back or anything I'm just thinking we want to use it with care.